Say Their Names/Source/View/Home/Person/PersonCell.swift

```diff
@@ -8,11 +8,28 @@
 
 import UIKit
 
-class PersonCell: UICollectionViewCell {
+final class PersonCell: UICollectionViewCell {
 
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var imgView: UIImageView!
+    @IBOutlet weak var dobLabel: UILabel!
+    @IBOutlet weak var bookmarkButton: UIButton!
+    
+    var data: Person? {
+        didSet {
+            //guard let data  = data else {return}
+        }
+    }
```

Comment:
In the future, you might need to set up the view as soon as you receive the object in this variable (which is better to rename it to `person` instead of `data`), but until there, you must avoid having unused commented code.

```suggestion
    var person: Person?
```

Reply:
Yeh a lot of this was taken from previous implementations in the project - will wait for direction to be made on coded layouts before updating.